ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp

```diff
@@ -152,9 +152,11 @@ std::vector<Tensor> _rdiv_bw(
 
 // unary_pow:
 // grad_input = grad * exponent * torch.pow(input, exponent - 1)
-std::vector<std::optional<Tensor>> _pow_bw(uint8_t queue_id, const Tensor& grad, const Tensor& input, float exponent, const MemoryConfig& output_mem_config, const std::vector<bool>& are_required_outputs, std::optional<Tensor> input_grad) {
+std::vector<std::optional<Tensor>> _pow_bw(uint8_t queue_id, const Tensor& grad, const Tensor& input, float exponent, const MemoryConfig& output_mem_config, std::optional<Tensor> input_grad) {
     std::vector<std::optional<Tensor>> grad_tensor;
-    TT_FATAL(are_required_outputs.at(0) , "input_grad derivative is required output");
+    if(!input_grad.has_value()){
+        input_grad = input;
+    }
```

Comment:
since this code is being duplicated for most ops, can we have a single common function that does this ?